src/pages/sign-in/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { signIn, getProviders, useSession, signOut } from "next-auth/react";
+import Head from "next/head";
+import { PageLayout } from "~/components/Layout";
+import {
+  type GetServerSidePropsContext,
+  InferGetServerSidePropsType,
+} from "next";
+import { PeerPrepRectLogo } from "~/assets/logo";
+import LoginWithCredentials from "~/components/Login";
+import Link from "next/link";
+
+// Signin page with credentials & oauth provider sign in
+type SignInProps = InferGetServerSidePropsType<typeof getServerSideProps>;
+const SignIn = ({ providers }: SignInProps) => {
```

Comment:
Why is the background for the pages black with white stripes, are we going to change it to a different color?

Reply:
Also have some overflow problems when scrolling
![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g39/assets/70463178/bb91aa1d-a641-4a7a-ad84-786461bde242)

Reply:
:( white stripes? thats not intended >< im still trying to change base css class to black but looking at the sass stuff i held back on that, if base css is black wont have patchy effects on loading

Reply:
thx for mentioning the overscrolling, ill have it the back of my head and try fix it in a diff pr?

Reply:
I meant the white stripes caused by border-x in pagelayout. Is that intended?

Reply:
It shouldn't be too hard to fix in this pr? I think it is caused by the padding of the sigup component
![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g39/assets/70463178/0ab8f21e-2a6c-4f8d-92d0-b6079730b08c)
Causing it to grab more space than supposed

> thx for mentioning the overscrolling, ill have it the back of my head and try fix it in a diff pr?